bebot/include/bebot/bebot.hpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <Eigen/Dense>
+
+namespace bebot {
+    class BeBOT {
```

Comment:
BeBOT is the name of the library but you probably don't want to name this class that. Maybe something like `bebot::ControlProblem` (this class describes a control problem to solve) or `bebot::TrajectorySolver` (this class solves for your trajectory), etc